integrationtests/build.gradle.kts

```diff
@@ -1,14 +1,15 @@
 plugins {
-    kotlin("jvm")
+    buildsrc.convention.`kotlin-jvm`
 }
 
 dependencies {
     testImplementation(Deps.jetbrainsAnnotations)
+    // TODO figure out if projects.project can be used instead of this format
     testImplementation(project(path = ":test-utils", configuration = "default"))
```

Comment:
I've not seen this way of defining project dependencies before. I presume I can replace it with `testImplementation(projects.testUtils)`?